### DISCUSSION TITLE
Saving Files!

### DISCUSSION THREAD

**Discussion by vicsidious** [darthcloud/BlueRetro#654] [Q&A] [answered]
Hi, I've been meaning to ask: Now that we can save per-console and per-game configurations into files stored in the BlueRetro filesystem, is there anyway to export those files to disk? Currently, the only option we are given is to delete files, but it would be nice to be able to export them, say before a firmware update, and then import them once again! 

If there is already a way, I have missed it completely :) 

Thanks all! 
-David

**Comment by darthcloud** [comment, marked as answer]
Not yet, it's probably not much work but touching the web config is not my favorite thing to work on.

Will be done one day for sure.
